### PR TITLE
Install Page Update

### LINKED ISSUE #1148: Outdated Documentation for Supported Platforms
The file [`supported.rst`](https://github.com/pyne/pyne/blob/develop/docs/install/supported.rst) could use some updating. Since all of the listed OS's have reached end of life, I doubt we still build and test PyNE on these platforms. I'm not sure where to find a complete list of the builds that we test but I found a short [list](https://github.com/pyne/pyne/blob/develop/.circleci/config.yml) in `.circleci` containing python2 with/without PyMOAB/DAGMC and python3 without PyMOAB and DAGMC, all built on ubuntu 18.04.

Additionally, the information on the binaries seems to be outdated as well. Viewing the conda [site](https://anaconda.org/conda-forge/pyne/files) suggests that we have binaries for Mac and Linux with releases 5.11, 5.10, 5.3, and 5.1.

### PR DESCRIPTION
**Pull request by nsryan2**
This PR updates the installation page of the website with the information @mehmeturkmen and I put together. It adds information about the Ubuntu scripts, adds a link to the How to test guide, and removes the "Create a Package (For Developer)" instructions on the conda install page (conda-recipes has been depreciated).

### CODE REVIEW

**Comment by nsryan2** [comment]
~~Sorry about the messy history, I accidentally compromised my default and had to make a backup.~~

Edit: Only the last three commits are relavent to this PR

File: docs/install/ubuntu.rst [hunk 1]
```diff
@@ -7,3 +7,49 @@ Ubuntu build script
...
+----------------------------
+
+The user may choose either
+to build a stable version of PyNE ("-stable") or the current develop
+branch of PyNE ("-dev") for each.
+
+Example for building a docker image of the latest stable branch of PyNE based on
+Ubuntu 16.04::
+
+    docker build -f ubuntu_16.04-stable.dockerfile -t pyne-16.04-stable .
```
Comment (kkiesling) [inline docs/install/ubuntu.rst]:
```suggestion
    docker build -f ubuntu_16.04-stable.dockerfile -t pyne-ubuntu-16.04-stable .
```

File: docs/install/ubuntu.rst [hunk 2]
```diff
@@ -7,3 +7,49 @@ Ubuntu build script
...
+-------------------
+Bash Scripts (*.sh)
+-------------------
+
+The script used to install PyNE should correspond
+to the user's operating system and version.
+
+The intention of these
+scripts is that PyNE can be ready to use on a clean install of any of
+the supported operating systems. Furthermore, the user should choose either
```
Comment (kkiesling) [inline docs/install/ubuntu.rst]: Can we link "supported operating systems" to the page that lists this? (or list them here if there is no page)

File: docs/install/ubuntu.rst [hunk 3]
```diff
@@ -7,3 +7,49 @@ Ubuntu build script
 Scripts for installing PyNE and all its dependencies from scratch on Ubuntu
 14.04 - 15.04 are found `here
```
Comment (gonuke) [inline docs/install/ubuntu.rst]:
```suggestion
16.04 & 18.04 are found `here
```
Is anyone game for adding 20.04?

**Review by gonuke — CHANGES_REQUESTED** [review]
We need to remove references to Windows from the `index.rst`.

Also a philosophical question: Should the instructions for using the build scripts be in the README of the install_scripts repo and then we just link to there?  Pro: PR's to install_scripts can be reviewed for updates to those instructions to ensure consistency. Con: they will not be formatted nicely as part of the website.

File: docs/install/ubuntu.rst (hunk 2 above)
Comment (nsryan2) [reply docs/install/ubuntu.rst]: Yes we could, however that page still needs to be updated. I'll add those updates to this PR, and then link it here

**Comment by nsryan2** [comment]
> Also a philosophical question: Should the instructions for using the build scripts be in the README of the install_scripts repo and then we just link to there? Pro: PR's to install_scripts can be reviewed for updates to those instructions to ensure consistency. Con: they will not be formatted nicely as part of the website.

After thinking about it some more, it would probably be best to just link to the readme here. This way, if someone went straight to the install scripts they wouldn't miss the instructions or have to worry about differences between two sets of instructions.

**Comment by bam241** [comment]
this need to be rebased and pushed

**Comment by opotowsky** [comment]
Hey @nsryan2 It has been brought to my attention that [openMC](https://docs.openmc.org/en/stable/index.html) is an optional dependency listed nowhere that I can see. 

Could you add it to optional dependencies on `docs/install/linux_source.rst` and the main pyne `readme.rst`?

**Comment by opotowsky** [comment]
> Hey @nsryan2 It has been brought to my attention that [openMC](https://docs.openmc.org/en/stable/index.html) is an optional dependency listed nowhere that I can see.
> 
> Could you add it to optional dependencies on `docs/install/linux_source.rst` and the main pyne `readme.rst`?

This would also maybe affect the install scripts, but someone else can speak to that (I think some do install the optional dependencies)

**Comment by bam241** [comment]
> > Hey @nsryan2 It has been brought to my attention that [openMC](https://docs.openmc.org/en/stable/index.html) is an optional dependency listed nowhere that I can see.
> > Could you add it to optional dependencies on `docs/install/linux_source.rst` and the main pyne `readme.rst`?
> 
> This would also maybe affect the install scripts, but someone else can speak to that (I think some do install the optional dependencies)

we don't support openMC in the install script for now...

**Comment by nsryan2** [comment]
> we don't support openMC in the install script for now...

@opotowsky Is openMC for the testing? If so, should I just add a note on the how to test pyne page that you need it to run the tests?

**Comment by kkiesling** [comment]
> > we don't support openMC in the install script for now...
> 
> Is openMC for the testing? If so, should I just add a note on the how to test pyne page that you need it to run the tests?

If you don't have OpenMC the tests should just be skipped I think? I don't think OpenMC needs to be listed in these files if I am processing what this PR is for. I think it just needs to be added to the dependency list as an optional dependency just like MOAB and DAGMC.

File: docs/install/linux_source.rst [hunk 4]
```diff
@@ -25,6 +25,7 @@ PyNE has the following dependencies:
 Optional Depenendencies:
    #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
    #. `PyTAPS <https://pythonhosted.org/PyTAPS/index.html>`_
```
Comment (kkiesling) [inline docs/install/linux_source.rst]: PyTAPS needs removed here and DAGMC needs to be added (we should probably note that DAGMC requires MOAB?)

**Review by kkiesling — CHANGES_REQUESTED** [review]

File: readme.rst [hunk 5]
```diff
@@ -41,6 +41,7 @@ PyNE has the following dependencies:
 Optional Depenendencies:
    #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
    #. `PyTAPS <https://pythonhosted.org/PyTAPS/index.html>`_
```
Comment (kkiesling) [inline readme.rst]: same thing- PyTAPS to be removed, DAGMC needs to be added

**Review by kkiesling — CHANGES_REQUESTED** [review]
It won't let me add specific line comments on non changed lines, but in the `readme.rst` there needs to be a few more changes:

* line 66 says `where VERSION should be replaced with the version number to be installed.` but there is no `VERSION` in the command
* the next paragraph talks about a windows binary but we don't support that so we should just remove that entire paragraph.

File: docs/install/supported.rst [hunk 6]
```diff
@@ -4,22 +4,12 @@
...
-on Python 2.7
+PyNE is currently built and tested on the following platforms
+on Python 3.6, 3.7, and 3.8:
 
- * Ubuntu 12.04 - x86_64
- * Ubuntu 10.04 - x86_64
- * Scientific Linux 6 - x86_64
- * OSX 10.8 - x86_64
+ * Ubuntu 16.04 - x86_64
+ * Ubuntu 18.04 - x86_64
```
Comment (mehmeturkmen) [inline docs/install/supported.rst:12]: We can also add Ubuntu 20.04 - x86_64 here as a supported platform. I installed pyNE from source and ubuntu_18.04.sh file. It is working with no problem.

**Review by mehmeturkmen — CHANGES_REQUESTED** [review]

File: docs/install/linux_source.rst (hunk 4 above)
Comment (nsryan2) [reply docs/install/linux_source.rst]: The link that I provided for DAGMC takes you to the page that encompasses both the link to the dependencies and the link to the installation instructions, which I think should do the trick. If I'm not mistaken, we have other dependencies that rely on each other in this way and we don't specify for those.

Comment (kkiesling) [reply docs/install/linux_source.rst]: ya true- probably not necessary to note here then

**Review by gonuke — APPROVED** [review]

**Review by kkiesling — APPROVED** [review]

**Pull request merged**